pmd-core/src/main/java/net/sourceforge/pmd/cli/PMDParameters.java

```diff
@@ -217,6 +217,9 @@ public PMDConfiguration toConfiguration() {
         LanguageVersion languageVersion = LanguageRegistry
                 .findLanguageVersionByTerseName(this.getLanguage() + ' ' + this.getVersion());
         if (languageVersion != null) {
+            if (language != null) {
```

Comment:
@adangel and @oowekyala do you think this will negative effects on existing PMD users? Not sure.